### PR TITLE
Fix checkbox appearance check

### PR DESCRIPTION
**Pull request by btecu**

### CODE REVIEW

File: src/api/form/PDFForm.ts [hunk 1]
```diff
@@ -579,6 +579,9 @@ export default class PDFForm {
         const ap = widget.getAppearanceCharacteristics();
         const rectangle = widget.getRectangle();
         const rotation = degrees(ap?.getRotation() ?? 0);
+        if (rectangle.height < 0) {
+          rectangle.y += rectangle.height;
+        }
```
Comment (btecu) [inline src/api/form/PDFForm.ts:584]: Should handle negative `y` as well:
```js
          rectangle.y = rectangle.y < 0
            ? rectangle.y - rectangle.height
            : rectangle.y + rectangle.height;
```

**Comment by Hopding** [comment]
@btecu apologies for my late response. I've been very busy these past few months taking care of some personal stuff and starting my new job at GitHub 🙂 

Is this PR still valid? If so, you'll need to either update this PR (via a force push) or open a new PR with your changes. The tl;dr is that I recently had to do a fairly extensive git history rewrite that caused a bunch of conflicts. Please read https://github.com/Hopding/pdf-lib#git-history-rewrite for details. Apologies for the hassle!

**Comment by btecu** [comment]
@Hopding I pushed to the branch but it doesn't seem to be updated here: https://github.com/Hopding/pdf-lib/compare/master...btecu:radiogroup

Also, it's been a while and I don't remember exactly which edge cases the changes we handling.

**Comment by Hopding** [comment]
Hmm. I'm able to view the diff without trouble, but GitHub still seems to think there are conflicting files for some reason.

**Comment by Hopding** [comment]
After reviewing the diff, I'm not sure I understand what scenarios this change solves for. As far as I can tell the current implementation seems to match the spec. Can you provide any additional context/details? (I'm guessing not since it's been so long).

**Comment by btecu** [comment]
I'll close this for now.

**Pull request closed without merging**